go.mod

```diff
@@ -9,25 +9,19 @@ require (
 	github.com/PuerkitoBio/purell v1.1.0 // indirect
```

Comment:
Please can I ask did you use go mod tidy to update this file and the go.sum file? I only ask because I noticed these two files changed when I run the code locally but I was unsure if to include them in the commit.

Reply:
This repo does have a strange issue with the server import that we use to run this in local dev. Because the makefile `go get`s that library it gets added, but shouldn't as it isn't a dependency of the actual script. That is a concern for another day though.